vendor/k8s.io/kubernetes/test/e2e/storage/testsuites/volume_expand.go

```diff
@@ -203,7 +203,15 @@ func (v *volumeExpandTestSuite) DefineTests(driver TestDriver, pattern testpatte
 			l.resource.Pvc = npvc
 
 			ginkgo.By("Creating a new pod with same volume")
-			l.pod2, err = e2epod.CreateSecPodWithNodeSelection(f.ClientSet, f.Namespace.Name, []*v1.PersistentVolumeClaim{l.resource.Pvc}, nil, false, "", false, false, e2epv.SELinuxLabel, nil, l.config.ClientNodeSelection, framework.PodStartTimeout)
+			newPodStartTimeout := framework.PodStartTimeout
```

Comment:
I'd suggest PodStartAfterResizeTimeout and have it the same for all volume plugins / CSI drivers. Double the original PodStartTimeout?

Reply:
done